libs/ha/ha_common_libs_k8s.py

```diff
@@ -781,9 +781,10 @@ def check_cluster_status(self, pod_obj, pod_list=None):
                 command_suffix=f"-c {common_const.HAX_CONTAINER_NAME} -- "
                                f"{common_cmd.MOTR_STATUS_CMD}", decode=True)
             for line in res.split("\n"):
-                if "failed" in line or "offline" in line or "unknown" in line:
-                    LOGGER.error("Response for data pod %s's hctl status: %s", pod_name, res)
-                    return False, f"Cortx HCTL status has Failures in pod {pod_name}"
+                if 'm0_client' not in line:
```

Comment:
Please try to use assert

Reply:
Assert is called in the caller function, which takes decision based on the returned value from this function.